### PR TITLE
Updates for unstable v3

### PR DESCRIPTION
**Pull request by OR13**
Now that we have https://w3id.org/security/v3-unstable

We need to cleanup the unstable definitions ASAP.

- add missing JsonWebKey2020 term
- add Ed235519Signature2020
- add Ed25519VerificationKey2020

### CODE REVIEW

File: index.html
Comment (tplooker) [inline index.html:738]: Good idea I will file a seperate PR to add the same to Bbs suites

File: contexts/security-v3-unstable.jsonld
Comment (tplooker) [inline contexts/security-v3-unstable.jsonld:39]: Just as a syntax note @dlongley @msporny do we need to duplicate this in nested context definitions or does what is defined at the root of the context propergate?

**Review by tplooker — APPROVED** [review]
LGTM but still getting my head around understanding JSON-LD context definitions

File: index.html
Comment (gjgd) [inline index.html:41]: nit: need indentation

File: index.html
Comment (gjgd) [inline index.html:628]: nit: indendation

File: contexts/security-v3-unstable.jsonld
Comment (gjgd) [inline contexts/security-v3-unstable.jsonld:9]: To be consistent with the syntax in previous security contexts, how about: `"JsonWebKey2020": "sec:JsonWebKey2020"`

Two benefits:
- uses the shorthand "sec" defined [here](https://w3c-ccg.github.io/security-vocab/contexts/security-v1.jsonld) instead of the full url "https://w3id.org/security#"
- more compact syntax with the implied "@id"

Thoughts?

Comment (OR13) [reply contexts/security-v3-unstable.jsonld:9]: https://github.com/w3c-ccg/security-vocab/issues/57

its not documented anywhere yet, but I think we will be adding lint rules warning people to avoid them.

File: index.html
Comment (OR13) [reply index.html:628]: yep, these are all over the place... imo the entire spect should be linted & formatted before each push.

File: contexts/security-v3-unstable.jsonld
Comment (dlongley) [reply contexts/security-v3-unstable.jsonld:11]: +1 to stopping recursive inclusion and using scoped contexts.

**Comment by OR13** [comment]
@msporny would you be opposed to merging this, and then,  https://github.com/w3c-ccg/security-vocab/pull/58

and then I will take a stab at removing `version` and using scoped contexts in a smaller PR that won't hold up tobias?

File: contexts/security-v3-unstable.jsonld
Comment (OR13) [reply contexts/security-v3-unstable.jsonld:20]: Opened https://github.com/w3c-ccg/security-vocab/issues/60 prefer to address in subsequent PR.

File: contexts/security-v3-unstable.jsonld
Comment (OR13) [reply contexts/security-v3-unstable.jsonld:11]: Opened https://github.com/w3c-ccg/security-vocab/issues/61 prefer to tackle in subsequent PR.

**Comment by OR13** [comment]
@msporny when you request changes, can you make it clear what needs to be changed?

something like....

blocking feedback: ....

nonblocking feedback: ...

This PR is already large, I'd prefer to tackle the issues you raised after merging this and #58.

**Comment by OR13** [comment]
@msporny can I get another review of this? I don't know how to implement changes to meet your approval.

**Comment by OR13** [comment]
Related issues, opened, changes are to unstable, the PR is holding up others that are approved. merging.

**Pull request merged**
